README.md

```diff
@@ -1,23 +1,27 @@
 # Calico Networking for rkt
 
 ## Building the plugin
-To build the calico-kubernetes plugin, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
+
+To build the Calico Networking Plugin for Rkt, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
 
 ## Using rkt Plugins
+
 The [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins in rkt will walk you through the basics of setting up networking in rkt.
 
 ## Requirements
+
 * A working [etcd](https://github.com/coreos/etcd) service
-* Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a [docker container](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md). For now, you will need to run calico in docker before starting your rkt contianers. This can be easily done by downloading the [calicoctl binary](https://github.com/projectcalico/calico-docker/releases) and running the following command: `calicoctl node --ip=$IP`
+* Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a [docker container](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md). For now, you will need to run calico in docker before starting your rkt contianers. This can be easily done by downloading the [calicoctl binary](https://github.com/projectcalico/calico-docker/releases) and running the following command: `sudo calicoctl node --ip=<IP> --rkt`
```

Comment:
A few things here:
- `calicoctl` is a requirement on its own, and you explain running the `node` command below already.
- Currently, Docker is also a requirement, and I think you can keep it simple and just have a note saying that support for running calico/node in Rkt containers is in progress, but for now Docker can be used to get it working.
- (This is in a sentence you'll probably remove, but..) Calico is always capitalized when referring to the product, and typo with 'rkt contianers'